File: languages/BTL/Haskell/Language/BTL/Universal/Sample.hs
-- BEGIN ...
module Language.BTL.Universal.Sample where
import Data.TermRep
-- END ...
sampleExpr :: TermRep
sampleExpr =
  TermRep "pred" [
    TermRep "if" [
      TermRep "iszero" [TermRep "zero" []],
      TermRep "succ" [TermRep "succ" [TermRep "zero" []]],
      TermRep "zero" [] ]]

File: languages/BTL/Java/org/softlang/btl/Sample.java
// BEGIN ...
package org.softlang.btl;
import org.softlang.btl.Syntax.*;
public class Sample {
// END ...
    static final Expr sample =
      new Pred(
        new If(
          new IsZero(new Zero()),
          new Succ(new Succ(new Zero())),
          new Zero()));
// BEGIN ...
}
// END ...
